src/components/Requirement.vue

```diff
@@ -3,36 +3,44 @@
     class = "requirement"
     :draggable = "canDrag"
     v-on:dragstart = "dragStart"
-    @click = "clickRequirement"
+    @mouseover = "hoveringOver = true"
+    @mouseleave = "hoveringOver = false"
   >
-    <div v-if="!leaf">
-      <span v-if="'title-no-degree' in req && req['title-no-degree'] !=''">{{ req["title-no-degree"] }}</span>
-      <span v-else-if = "'short-title' in req && req['short-title'] != ''">{{ req['short-title']}}</span>
-      <span v-else-if = "'title' in req">{{ req["title"] }}</span>
-      <span style="font-style:italic">{{ req['threshold-desc'] }}</span>
-      <!--fake padding for scroll-->
-      &nbsp &nbsp &nbsp
-      <v-btn icon flat fixed v-if = "showDelete" @click = "viewDialog = true;"><v-icon>delete</v-icon></v-btn>
-    </div>
-    <span v-else>
-      <span v-if="'title' in req">{{ req.title }}</span>
-    </span>
-
-    <span v-if = "!req['plain-string']">
-      <span v-if="!('title' in req) && 'req' in req">
-        <span :class="reqFulfilled">{{ req.req }}</span>
-        <span style = "font-style:italic" v-if = "'threshold-desc' in req">({{ req['threshold-desc']}})</span>
-      </span>
-    </span>
-    <span v-else>
-      <span v-if = "'title' in req">| </span>
-      <span style = "text-transform: cursive">{{ req.req }}</span>
-    </span>
-    <span v-if = "req.max === 0 && leaf" style = "font-style:italic">
-       (optional)
-    </span>
-    <div :class = "percentage_bar" :style = "percentage"></div>
+  <v-layout row>
+    <v-flex>
+      <div>
```

Comment:
do we need this div? I tested removing it and it didn't seem to break anything, but I assume it's here for a reason?

Reply:
Yeah, I can't remember why that div was there but it's obviously no longer necessary.